pkgs/by-name/ph/phraze/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "phraze";
+  version = "0.3.10";
+
+  src = fetchFromGitHub {
+    owner = "sts10";
+    repo = "phraze";
+    rev = "v${version}";
+    hash = "sha256-Kg3s5iNQajGFT1+YJFdCvq6UoFJS/YDfhdAOp6xV8w4=";
+  };
+
```

Consider enabling tests, they seem reasonable.

```suggestion

  doCheck = true;

```